src/components/Header.tsx

```diff
@@ -152,11 +152,11 @@ const SignUp = styled.button`
   margin: 0 15px;
 `
 
-const Header: React.FC = () => {
+export default function Header() {
```

Comment:
prefer regular function over arrow

Reply:
Great question. It's just a preference. I do have some backups for it tho:

- no additional advantage using arrow func
- no need to isolate (no hoist), which unnecessarily forces to order funcs from top to bottom
- it actually makes the code longer
- better readability for `function` than `const`

```ts
const Header = () => {}

function Header() {}
```

For exports:

- I prefer barrel exports
- I prefer named exports
- react components are less likely to have multiple exports in a single file
- it makes the code longer

```ts
const Header = () => {} export default Header

export default function Header() {}
```

At the end of the day, longer codes don't really matter, but I also try to avoid unnecessary additions.